moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -38,9 +38,14 @@
  *
  */
 
-#include <moveit_servo/servo_node.hpp>
+#if __has_include(<realtime_tools/realtime_helpers.hpp.hpp>)
```

Comment:
almost comically - you still have .hpp.hpp

Reply:
I think I need to hand back my college degree...

```suggestion
#if __has_include(<realtime_tools/realtime_helpers.hpp>)